### ISSUE TITLE
New feature: close side panel manually

### ISSUE THREAD

**Issue by scramatte** [dcasia/collapsible-resource-manager#120]
Hello,

Can you add button to collapse side panel manually?
When you  choose a menu item, the side menu remain opened,  that is fine  but it would be cool to be able to close it manually to get more space.

Regards

**Comment by milewski** [comment]
When you load the page it will be expanded by default when there is a menu selected, if you click on the same menu again.. it will close .. inst this a manual button already?

**Comment by vitorcrovador** [comment]
It doesn't happen like that here. Even if I click on the selected menu button it doesn't close.

It would be very interesting if there was a parameter for the initial default state to be expanded or closed.
And the side panel closes when clicking anywhere else on the screen.

> When you load the page it will be expanded by default when there is a menu selected, if you click on the same menu again.. it will close .. inst this a manual button already?

**Comment by vitorcrovador** [comment]
Yesterday, trying to find out why closing the menu wasn't working, for some reason I still don't know what it is, it started working. However, after doing a composer update it stopped working again (without adding or removing any packages).
I hope that in some way this information helps to correct the error.

**Comment by milewski** [comment]
Could that be the nova version? are you running on the latest version?

**Comment by vitorcrovador** [comment]
> Could that be the nova version? are you running on the latest version?

Yes, I´m using the latest version (v4.27.14)

**Comment by vitorcrovador** [comment]
I think I found where the error is, I just don't know how to fix it.
It's in the setActiveMenu function. I did a direct override in the browser to debug and it seems that the currentActiveMenu variable is not the same as the "menu" variable because it comes as "Proxy Object".
I sent a print, I believe that this way you will be able to understand and correct it.
![error-collpsible-menu](https://github.com/dcasia/collapsible-resource-manager/assets/22171591/fd94a2ab-bd63-4f40-8632-75c817c20d7b)
